doc/historylog.rst

```diff
@@ -4,11 +4,9 @@ History
 
 With the **History** pane, you can view all the commands you've entered into any :doc:`ipythonconsole`, along with their timestamp.
 
-.. image:: images/history/history-menu.png
+.. image:: images/history/history-standard.png
```

Comment:
This screenshot looks good, but `optipng` reduces the size of this 0.5 MB image by nearly 50%, entirely for free (aside from a few seconds of runtime). I can do it while rebasing before the merge, if desired.

Reply:
Will be addressed on rebase